src/graphql/trusted_domain.rs

```diff
@@ -62,6 +62,29 @@ impl TrustedDomainMutation {
         Ok(name)
     }
 
+    /// Update a trusted domain, returning the new value if it passes domain validation.
```

Comment:
Since this API does not run validation, I think the documentation comment needs to be modified.